apps/editing-toolkit/editing-toolkit-plugin/readme.txt

```diff
@@ -1,5 +1,5 @@
 === WordPress.com Editing Toolkit ===
-Contributors: alexislloyd, allancole, automattic, bartkalisz, codebykat, copons, dmsnell, get_dave, glendaviesnz, gwwar, iamtakashi, iandstewart, jeryj, Joen, jonsurrell, kwight, marekhrabe, mattwiebe, mkaz, mmtr86, mppfeiffer, noahtallen, nosolosw, nrqsnchz, obenland, okenobi, owolski, philipmjackson, vindl
+Contributors: automattic
```

Comment:
I wouldn't use `automattic` because these should be wordpress.org usernames and that user doesn't exist.

Removing the field entirely causes a warning in the [validator](https://wordpress.org/plugins/developers/readme-validator/), which might be fine?

[Readme docs](https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/)
[Readme validator](https://wordpress.org/plugins/developers/readme-validator/)

Reply:
It was already on the list, and it does exist: https://profiles.wordpress.org/automattic/

See https://wordpress.org/plugins/full-site-editing/

<img width="750" alt="Screenshot 2021-01-22 at 14 23 17" src="https://user-images.githubusercontent.com/87168/105490437-70719b80-5cbd-11eb-863b-5970bf2e8643.png">

Reply:
TIL about the readme validator! 😮

Reply:
🤦  I got the URL wrong when looking for it. `automattic` it is 👍